pythonforandroid/recipes/ffmpeg/__init__.py

```diff
@@ -4,8 +4,8 @@
 
 
 class FFMpegRecipe(Recipe):
-    version = '3.4.5'
-    url = 'http://ffmpeg.org/releases/ffmpeg-{version}.tar.bz2'
+    version = 'n3.4.5'
+    url = 'https://github.com/FFmpeg/FFmpeg/archive/{version}.zip'
```

Comment:
shouldn't we have the `n` here `https://github.com/FFmpeg/FFmpeg/archive/n{version}.zip` and then removed from the version?

Reply:
Mmm, IMHO We should have the `n` in the version, so, if the user wants to use a specific version / hash from requirements he didn't have to edit also the ffmpeg recipe.

Reply:
OK :+1: 
Would you mind adding a comment above the `url` maybe, specifically stating we're using github over ffmpeg.org for performance reasons. That way we don't revert it later if we forget why it's like that

Reply:
Sure! Added.